Week1/homework/app.js

```diff
@@ -1,11 +1,141 @@
 'use strict';
+//1.1
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+const bookTitles = [
+  // Replace with your own book titles
+  'harry_potter_chamber_secrets',
+  'the_selfish_gene',
+  'thinking_fast_slow',
+  'the_idiot_brain',
+  'the_laws_of_human_nature',
+  'how to analyze people',
+  'how_to_win_friends',
+  'visual_thinking',
+  'think_data_structure',
+  'list_book',
+];
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+console.log(bookTitles);
+
+//1.2
+
+console.log(bookTitles[2]);
+
+//1.3
+let button = document.getElementById('clickButton');
+
+button.addEventListener('click', function() {
+  const divElement = document.createElement('div');
+
+  document.getElementsByClassName('flexbox1')[0].appendChild(divElement);
+  let list = document.createElement('ul');
+  divElement.appendChild(list);
+  let line = document.createElement('li');
+  line.innerText = 'harry_potter_chamber_secrets';
```

Comment:
Please read the instructions again carefully. The idea is that you put all the data about books in the `bookDetails` object and that you loop over the `bookTitles` array to get the data from this object.
You have now put the data (like the title) in function instead of taking it from the object.

Reply:
I changed this code. Created new for loop inside of the function. But there is not any bookDetails object yet. I picked bookTitles's child. 

//1.3

Thank you for your feedback